packages/hash/api/src/collab/Instance.ts

```diff
@@ -619,6 +701,10 @@ export class Instance {
    * @todo do we want to force sending updates here too?
    */
   private addUpdates(updates: Update[]) {
+    if (this.errored) {
+      return false;
```

Comment:
Same as `updateSavedContents`, the return here makes it so both cases of the function's return type would be falsy